message/validation/errors.go

```diff
@@ -52,31 +52,36 @@ func (e Error) Text() string {
 }
 
 var (
-	ErrGenesisSSVMessage                       = Error{text: "genesis ssv message"}
-	ErrGenesisSignedSSVMessage                 = Error{text: "genesis signed ssv message"}
-	ErrEmptyData                               = Error{text: "empty data"}
 	ErrWrongDomain                             = Error{text: "wrong domain"}
 	ErrNoShareMetadata                         = Error{text: "share has no metadata"}
 	ErrUnknownValidator                        = Error{text: "unknown validator"}
 	ErrValidatorLiquidated                     = Error{text: "validator is liquidated"}
 	ErrValidatorNotAttesting                   = Error{text: "validator is not attesting"}
-	ErrEarlyMessage                            = Error{text: "early message"}
-	ErrLateMessage                             = Error{text: "late message"}
-	ErrTooManySameTypeMessagesPerRound         = Error{text: "too many messages of same type per round"}
-	ErrRoundTooHigh                            = Error{text: "round is too high for this role" /*, reject: true*/} // TODO: enable reject
+	ErrEarlySlotMessage                        = Error{text: "message was sent before slot starts"}
+	ErrLateSlotMessage                         = Error{text: "current time is above duty's start +34(committee and aggregator) or +3(else) slots"}
+	ErrSlotAlreadyAdvanced                     = Error{text: "signer has already advanced to a later slot"}
+	ErrRoundAlreadyAdvanced                    = Error{text: "signer has already advanced to a later round"}
+	ErrDecidedWithSameSigners                  = Error{text: "decided with same number of signers"}
+	ErrPubSubDataTooBig                        = Error{text: "pub-sub message data too big"}
```

Comment:
why is this not reject? how could someone send a too big message and it'll be ok?
I can think of a fork scenario where message size decreases and we don't want to reject nodes that haven't updated yet. but this is only a short timespan.

@nkryuchkov @MatheusFranco99

Reply:
Aligned to the knowledge base: https://github.com/ssvlabs/knowledge-base-internal/blob/main/ssv/network/p2p/MessageValidation/Rules.md

cc @GalRogozinski